src/auth/local.strategy.ts

```diff
@@ -11,7 +12,10 @@ export class LocalStrategy extends PassportStrategy(Strategy) {
     });
   }
 
-  public async validate(email: string, password: string) {
+  public async validate(
+    email: string,
+    password: string
+  ): Promise<Omit<User, 'password'>> {
```

Comment:
Type "Omit<User, 'password'>" is duplicating. Make special type for that and use it

Reply:
done